docker/compose/docker-compose.postgres-tika.yml

```diff
@@ -77,7 +77,7 @@ services:
       PAPERLESS_TIKA_ENDPOINT: http://tika:9998
 
   gotenberg:
-    image: gotenberg/gotenberg:7.4
+    image: docker.io/gotenberg/gotenberg:7.4.4
```

Comment:
Is there a reason why this became locked to a more specific version as well?

Reply:
Oh - that was not on purpose. Thanks for catching this. I have upated the PR.